components/Chips/src/MDCChipField.m

```diff
@@ -57,6 +58,14 @@ @interface MDCChipTextField : MDCTextField
 
 @end
 
+@interface MDCTextField (Fundament)
```

Comment:
Please name the category to something prefixed with `MDCChips` or `MDCChipField` to avoid naming conflicts.

Suggestion: `@interface MDCTextField (MDCChipFieldVisibility)`

Reply:
Done